app/models/form_profile.rb

```diff
@@ -101,7 +101,8 @@ class FormProfile
     vre_readiness: ['28-1900'],
     coe: ['26-1880'],
     adapted_housing: ['26-4555'],
-    intent_to_file: ['21-0966']
+    intent_to_file: ['21-0966'],
+    ivc_champva: ['10-7959F-1']
```

Comment:
I think we may want to name this something specific to F1 rather than `ivc_champva`, because as we add other ivc forms they’ll have to have unique entries in this object